plugin/pkg/proxy/connect.go

```diff
@@ -152,9 +152,9 @@ func (p *Proxy) Connect(ctx context.Context, state request.Request, opts Options
 		rc = strconv.Itoa(ret.Rcode)
 	}
 
-	RequestCount.WithLabelValues(p.addr).Add(1)
-	RcodeCount.WithLabelValues(rc, p.addr).Add(1)
-	RequestDuration.WithLabelValues(p.addr, rc).Observe(time.Since(start).Seconds())
+	RequestCount.WithLabelValues(p.proxyName, p.addr).Add(1)
+	RcodeCount.WithLabelValues(p.proxyName, p.addr, rc).Add(1)
```

Comment:
The `RequestCount` and `RcodeCount` metrics seem to be duplicates of each other, as well as duplicating the `_count` value of the `RequestDuration` metric.

Let's drop them as they are just extra cardinality data.

Reply:
Have removed these and updated the `README.md` with pointers to derive the value from the `_count` value of the `requestDuration` metric.